File: src/main/java/com/hellosign/sdk/resource/support/types/RoleType.java
package com.hellosign.sdk.resource.support.types;

// Role codes are "a" for "admin" and "m" for "member"
public enum RoleType {
    ADMIN {
        @Override
        public String toString() {
            return "a";
        }
    },
    MEMBER {
        @Override
        public String toString() {
            return "m";
        }
    }
}

File: src/main/java/com/hellosign/sdk/resource/support/types/RoleType.kt
package com.hellosign.sdk.resource.support.types

import java.lang.IllegalArgumentException

// Role codes are "a" for "admin" and "m" for "member"
enum class RoleType(val value: String) {
    ADMIN("a"),
    MEMBER("m");

    override fun toString(): String {
        return value
    }

    companion object {
        @JvmStatic
        fun getEnum(value: String): RoleType? {
            return enumValues<RoleType>().find { it.value === value }
        }
    }
}
